src/autogluon/bench/scripts/generate_cloud_configs.py

```diff
@@ -0,0 +1,144 @@
+    module: str = typer.Option(..., help="The module to generate config for: 'multimodal' or 'tabular'"),
+    root_dir: str = typer.Option(".ag_bench_runs", help="Root directory (optional, default = '.ag_bench_runs')"),
+    benchmark_name: str = typer.Option("ag_bench", help="Benchmark name (optional, default = 'ag_bench')"),
+    cdk_deploy_account: str = typer.Option(..., help="CDK deploy account"),
+    cdk_deploy_region: str = typer.Option(..., help="CDK deploy region"),
+    prefix: str = typer.Option(
+        "ag-bench", help="Prefix used to identify infra resources (optional, default = 'ag-bench')"
+    ),
+    metrics_bucket: str = typer.Option(..., help="Name of the metrics bucket (required)"),
+    data_bucket: str = typer.Option(None, help="S3 bucket to download private datasets (optional)"),
+    vpc_name: str = typer.Option(None, help="Existing VPC name (optional, default: create a new VPC)"),
+    max_machine_num: int = typer.Option(
+        None, help="Maximum number of machines for AWS Batch (optional, default = 20)"
+    ),
+    block_device_volume: int = typer.Option(
+        None, help="Block device volume for EC2 instances (optional, default = 100)"
+    ),
+    reserved_memory_size: int = typer.Option(
+        None, help="Reserved memory size for Docker container (optional, default = 15000)"
+    ),
+    instance: str = typer.Option(None, help="EC2 Instance type (optional, default = 'g4dn.2xlarge')"),
+    git_uri_branch: str = typer.Option("", help="AutoGluon MultiModal git_uri#branch (comma-separated)"),
+    dataset_names: str = typer.Option(
+        "",
+        help="AutoGluon MultiModal dataset names (comma-separated, to get a list of dataset names, run `from autogluon.bench.datasets.dataset_registry import multimodal_dataset_registry\n multimodal_dataset_registry.list_keys()`)",
+    ),
+    presets: str = typer.Option(
+        None,
+        help="AutoGluon MultiModal presets (comma-separated, can choose from ['medium_quality', 'high_quality', 'best_quality'])",
+    ),
+    time_limit: str = typer.Option(None, help="AutoGluon MultiModal time limit (comma-separated)"),
+    hyperparameters: str = typer.Option(
+        None,
+        help="AutoGluon MultiModal hyperparameters (in the format '\"key1:value1,key2:value2;key1:value1,key2:value2\"'). Refer to https://auto.gluon.ai/stable/tutorials/multimodal/advanced_topics/customization.html for hyperparameter cutomization.",
+    ),
+    framework: str = typer.Option("AutoGluon:stable", help="AMLB Framework, default = (comma-separated)"),
+    amlb_benchmark: str = typer.Option("", help="AMLB Benchmark (comma-separated)"),
+    amlb_task: str = typer.Option(
+        None, help="AMLB Task (in the format '\"benchmark1:task1,task2;benchmark2:task3,task4\"')"
+    ),
+    amlb_constraint: str = typer.Option("", help="AMLB Constraint (comma-separated)"),
+    amlb_custom_branch: str = typer.Option(None, help="AMLB Custom Branch (comma-separated)"),
```

Comment:
Same question regarding the comma-separated. I probably misunderstood, but some explanation could be helpful

Reply:
It actually refers to custom branch of AG. I wanted to prefix the help so that people know what to supply for different modules. But maybe it would be better if I have separate CLI commands for each, e.g. `agbench generate_multimodal_cloud_configs` and `agbench generate_tabular_cloud_configs`

Reply:
Nah, I think `generate_config --module` is good.